upgrade/prepare_for_upgrade.md

```diff
@@ -132,6 +132,7 @@ Return here after verifying that SNMP is properly configured on the management n
 
    > This [System Admin Toolkit (SAT)](../glossary.md#system-admin-toolkit-sat) command has `shutdown` as one of the command line options,
    > but it will **not** start a shutdown process on the system.
+   > *Note:* This step may take a longer time if there are many BOS sessions. It is recommended to keep the sessions count minimal to reduce the overall time taken.
```

Comment:
The way this gets rendered, it folds and wraps the lines in the quoted text, so the Note does not appear on its own line, and it is easy to miss. I would recommend perhaps putting this in a separate quoted block, so it is more clear. I think that can probably be achieved by adding a blank line between lines 134 and 135, but you'll have to check.

![Screen Shot 2024-02-08 at 5 55 32 PM](https://github.com/Cray-HPE/docs-csm/assets/77809410/65e7f631-ac05-48be-ab68-df841422e02e)